docs/modules/user-guide/partials/proc_adding-volume-components.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-volume_{context}"]
+= Adding `volume` components to a devfile
+
+[role="_abstract"]
+Use volume components in order to share files among container components, making it easier to collaborate with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+* Define a component using the type `volume`.
++
+.A minimal `volume` component
```

Comment:
Is this example the user's primary source of information about `volume` components? If so, you might want to cover any volume-related attributes in this example or topic, for example, by using callouts. Otherwise, consider providing a cross-reference to the reference topic where this information is available.

Reply:
Okay to use monospaced fonts in headings?

Reply:
Dev review: do you have suggestion to add to to this yaml sample to address Rolfe's points above?